plugin/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/index.jelly

```diff
@@ -71,7 +71,8 @@ THE SOFTWARE.
                     <input type="button" id="generatePipelineScript" value="${%Generate Pipeline Script}"
                            class="submit-button primary"
                            data-url="${rootURL}/${it.GENERATE_URL}"
-                           data-crumb="${h.getCrumb(request)}"/>
+                           data-crumb-header="${h.getCrumbRequestField()}"
+                           data-crumb-value="${h.getCrumb(request)}"/>
```

Comment:
these should already be set in the `head` element produced by the [`layout`](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/layout/layout.jelly#L104-L107) tag.
Do we need to set them here - or can we use what comes from the layout and use `document.head.getAttribute(...)`